deploy/chart/values-external.yaml

```diff
@@ -7,6 +7,7 @@ deploy:
     secretName: alibaba-addon-secret
     idKey: access-key-id
     secretKey: access-key-secret
+  newProvisionerToken: false
```

Comment:
Why do we need this? I think we are not issuing token to registered cluster anyway, neither old nor new one?